<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Forms</class>
 <widget class="QWidget" name="Forms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="DocForm">
      <attribute name="title">
       <string>Document Forms</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidgetDocForms"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="LabelForm">
      <attribute name="title">
       <string>Label Forms</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget_LabelForms"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
